apps/src/templates/sectionProgressV2/progress-table-v2.module.scss

```diff
@@ -148,11 +164,11 @@
     }
 
     &LevelCell {
+      // If width is updated, must also update `LEVEL_HEADER_WIDTH` in ExpandedProgressDataColumn.jsx
```

Comment:
Could you put this number in a shared constant referenced from both places?

Also I see you're using a `min-width` here and assuming that the actual width will be the same when setting the width of the header. Are there cases where the width would be more than this, or should we switch this `min-width` to a `width`?